recipes/treemaker/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "treemaker" %}
+{% set version = "1.1" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "f26b28a8effd759b4e187f7e10213b1c48b84c4bcd849f6acf5c96ff5c18ee80" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  entry_points:
+    - treemaker = treemaker:main
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  host:
```

Comment:
Not yet on cb3 even if `skeleton` thinks otherwise 😉

Reply:
I.e., need `build:` instead.

Reply:
thanks, bad habit of trying to fix a PR before running out of office.